libs/backend/intermediate-scripts/set-clock

```diff
@@ -30,4 +30,8 @@ else
     timedatectl set-timezone "${1}"
 fi
 
+# If the parent node process has a TZ environment variable set, it will
+# override the timezone set by timedatectl. So we unset it here.
+unset TZ
```

Comment:
I know the only caller sets `process.env.TZ` but what do you think about storing this so we can resassign the old value of `TZ` once the time change is done?

Reply:
As I was debugging this, I actually discovered that `TZ` is undefined on startup. I'm not sure where it's caching the timezone, maybe internally in the engine 🤷